src/pages/LandingPages/CorporateCampaign/CCLandingPage.vue

```diff
@@ -36,6 +36,8 @@
 				/>
 				<campaign-loan-wrapper
 					ref="mlLoanDisplay"
+					:promo-amount="promoAmount"
```

Comment:
These props should get passed in via the campaignLoanWrapperProps object instead of directly as a prop itself.